src/Packetizer/SpacePacket.h

```diff
@@ -0,0 +1,47 @@
+/**@file
+ * This file is part of the TASTE Runtime Common.
+ *
+ * @copyright 2021 N7 Space Sp. z o.o.
+ *
+ * TASTE Runtime Common was developed under a programme of,
+ * and funded by, the European Space Agency (the "ESA").
+ *
+ * Licensed under the ESA Public License (ESA-PL) Permissive,
+ * Version 2.3 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://essr.esa.int/license/list
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef SPACE_PACKET_H
+#define SPACE_PACKET_H
+
+#define SPACE_PACKET_PRIMARY_HEADER_SIZE_BYTES 6u
```

Comment:
This shall be public. Some other constants (which are checked) too.

Reply:
Done, SpacePacket.h is now a public header + created SpacePacketInternal.h for masks and offsets.

Reply:
Good